addons/project/static/src/views/widget/subtask_counter.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<templates>
+
+    <t t-name="project.SubtaskCounter" owl="1">
+        <a t-attf-title="{{ closedSubtaskCount }} sub-tasks closed out of {{ subtaskCount }}"
+           class="subtask_list_button btn-link text-dark"
+           t-on-click.prevent="() => this.onClick()">
+            <span class="fa fa-check-square-o me-1"/>
+            <t t-esc="closedSubtaskCount + '/' + subtaskCount"/>
```

Comment:
I feel like this is not good for translations.
Could you make it a translatable string as a property on the component?